en/docs/portal/transports/manage-transport.md

```diff
@@ -1,15 +1,14 @@
 # Manage Transport
 
-To display this page, click **Transports** in the left pane.
+Clicking **Transports** in the left pane displays the Transports page.
 
 ![null](</docs/resources/images/transports/manage-transports.png>)
 
+- To filter transports, type characters in the search box and then press the Enter key or click magnifier icon, the filtering is case insensitive. To remove the filter, click the **Reset** button.
```

Comment:
Please change "click magnifier icon, the" to "click the magnifier icon. The"
We need to start a new sentence instead of separating them with commas.